tools/generator/src/Extensions/XCScheme+Extensions.swift

```diff
@@ -103,10 +107,23 @@ extension XCScheme {
 
         let profileAction: XCScheme.ProfileAction?
         if let profileActionInfo = schemeInfo.profileActionInfo {
+            let scriptTitle: String
+            let scriptText: String
+            if buildMode == .xcode {
+                scriptTitle = "Update .lldbinit"
+                scriptText = XCScheme.ExecutionAction.createLLDBInitScript
+            } else {
+                scriptTitle = "Update .lldbinit and copy dSYMs"
+                scriptText = XCScheme.ExecutionAction.createLLDBInitScript +
+                    "\n" + XCScheme.ExecutionAction.copyDSYMsScript
+            }
```

Comment:
I made the change to only copy the dSYMs in BwB mode. A also merged the actions into one, since they have the same `buildableReference`.